src/login/translation/src/login/translation/defaultMessages/userProfile.json

```diff
@@ -63,6 +63,22 @@
     "id": "dashboard_nav.advanced-settings",
     "defaultMessage": "Advanced settings"
   },
+  {
+    "id": "dashboard_nav.identity-verify-freja",
```

Comment:
personally, I would avoid using the word "card" to avoid confusing Freja eID with a physical ID card

Reply:
For usability I have align the text with the "vetting card" description text in "Identity" 
If you think that this is confusing I suggest that we rename the "vetting card" from "With a digital id-card" to "with a Freja eID" and then align the text in the tips.

Reply:
I think we should just call it eID or Swedish eID as that seems to be the offical term.

Reply:
Ok, I can agree with "Swedish eID"
However, as I said, if we are to update this text we will have to update the vetting card and tips with the same description text.
If we can agree on the above text, I will update the text to "Swedish eID" in a separate branch.